README.rst

```diff
@@ -1,5 +1,5 @@
 **Flit** is a simple way to put Python packages and modules on PyPI.
-It tries to require less thought about packaging and help you avoid common
+It tries to require less thought about packaging and helps you avoid common
```

Comment:
"to require [...] and help" seems correct to me. "to require [...] and to help" is a bit redundant. The suggested "it tries to require [...] and [it] helps" can work, but I would not say it is better.